microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -491,11 +569,14 @@ def verify_btrfs_nvme_datadisk(self, log_path: Path, result: TestResult) -> None
     @TestCaseMetadata(
         description="""
         This test case will run cifs xfstests testing against
-         azure file share.
-
-        Downgrading priority from 3 to 5. The file share relies on the
-         storage account key, which we cannot use currently.
-        Will change it back once file share works with MSI.
+        azure file share.
+        The case will provision storage account with private endpoint
+        and use access key // ntlmv2 for authentication.
+        This will be changed to MSI in the near future
+        To modify the test case parameters:
+        Update the mount options via '_default_smb_mount'
```

Comment:
Don't mention internal variables in the description. The description is published separated without code. To improve readability, you can give a brief introduction on the options, include/excluded tests. If people are really interested on details, they can look at the code.